graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH7.kt

```diff
@@ -3,15 +3,16 @@ package com.anaplan.engineering.azuki.graphs.eacs
 import com.anaplan.engineering.azuki.core.runner.Eac
 import com.anaplan.engineering.azuki.core.system.BEH
 import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphFunctionalElements
 import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
 import com.anaplan.engineering.azuki.graphs.graphA
 
-@BEH(GraphBehaviours.HasCycle, GraphBehaviours.CreateDirectedGraph, """
-    Determine the number of cycles in graph
+@BEH(GraphBehaviours.HasCycle, GraphFunctionalElements.Cycle, """
+    Determine the number of cycles in a given directed graph
 """)
 class BEH7 : GraphScenario() {
 
-    @Eac("When a directed graph has one cycle, return 1")
+    @Eac("When a given graph has one cycle, return 1")
```

Comment:
💡 It might be worth having the definition of cycle as an AC, i.e. something like:

> A cycle is a continuous path which starts and ends at the same vertex

If you do this, BEH6 should have the same AC.